src/components/pages/Leaderboard/Leaderboard.css

```diff
@@ -0,0 +1,57 @@
+h2 {
+    margin: 0;
+}
+
+
```

Comment:
Some of the spaces between css elements are single and some are double spaced.  Stick to one convention. Otherwise looks good.